src/Template.php

```diff
@@ -299,7 +299,7 @@ public function getBlockNames(array $context, array $blocks = [])
     /**
      * @return Template|TemplateWrapper
      */
-    protected function loadTemplate($template, $templateName = null, $line = null, $index = null)
+    protected function loadTemplate(string|TemplateWrapper|array $template, string $templateName = null, int $line = null, int $index = null)
```

Comment:
I suggest adding `@param string|TemplateWrapper|array<string|TemplateWrapper> $template` in the phpdoc to document the supported types for array values.

Btw, it looks like `Environment::resolveTemplate` supports passing a Template in the array (while it is not supported as a single value). And `Template::loadTemplate` also has support for it just below (even though `Environment::load` does not). Should this be deprecated ?

Reply:
Done